README.md

```diff
@@ -28,13 +28,16 @@ Among other things, the Tekton Dashboard supports:
 
 ## Read the docs
 
-It is **strongly recommended** to use the **v0.6.1.4** release or newer for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4
+It is **strongly recommended** to use the **v0.6.1.4** release or newer for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4.
 - This is a **critically important security release**
 - Earlier versions are deprecated and should be used for **development or isolated usage only**
 
+The latest release is **v0.7.0**.
```

Comment:
Do we need to explicitly call this out or is it obvious enough from the table below?

Reply:
Gone, obvious enough